docs/identities/model/state.mdx

```diff
@@ -0,0 +1,37 @@
+---
+id: state
+title: Activate and deactivate identities
+slug: activate-deactivate-user-login-state
+sidebar_label: De-activate identities
```

Comment:
```suggestion
sidebar_label: Change identity state
```
I think it's a good compromise considering the fact that two actions are important here.

Reply:
State is pretty generic. I think if you were to look for "how do i disable a user (login)" in the docs, you might not look for "identity state". With De-activate identities or something similar, one would find this more easily in the sidebar imo

Reply:
Ok, I don't have anything better than "de-activate" so let's leave it at that.

Reply:
Ok, we can always revisit / change this if we have another better idea